<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>400</width>
      <height>510</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="TableLayout">
     <item>
      <widget class="QTableWidget" name="Table"/>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>331</width>
      <height>150</height>
     </rect>
    </property>
    <property name="title">
     <string>Поиск абитуриентов</string>
    </property>
    <widget class="QLineEdit" name="line3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>80</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="line1">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>20</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>46</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Имя</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SearchStudent">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>120</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Найти абитуриента</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>90</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Код абитуриента</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Фамилия</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>50</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>331</width>
      <height>121</height>
     </rect>
    </property>
    <property name="title">
     <string>Поиск договоров</string>
    </property>
    <widget class="QPushButton" name="SearchTeacher">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>130</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Найти договор</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер договора</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ФИО</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>180</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>530</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>19</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
